src/index.html

```diff
@@ -50,6 +50,7 @@ <h3>Sleep Hygiene</h3>
   <script type="text/javascript" src="../data/sleep.js"></script>
   <script type="text/javascript" src="../data/activity.js"></script>
   <script type="text/javascript" src="./UserRepository.js"></script>
+  <script type="text/javascript" src="./Hydration.js"></script>
```

Comment:
I guarantee you I would have forgotten to do this step so thank you for remembering!